src/languageServerInstaller.ts

```diff
@@ -33,6 +40,7 @@ export class LanguageServerInstaller {
 		}
 
 		try {
+			this.reporter.sendTelemetryEvent('installingLs', { version: currentRelease.version });
 			await this.installPkg(currentRelease, directory, userAgent);
 		} catch (err) {
 			vscode.window.showErrorMessage('Unable to install terraform-ls');
```

Comment:
Is it worth logging the installation exception here too?

Reply:
This one gets logged by the caller in `extension.ts`.

Reply:
I see 👍🏻